tests/averaging/translation/test_averaging_1dsfm.py

```diff
@@ -153,5 +154,74 @@ def test_pickleable(self):
             self.fail("Cannot dump rotation averaging object using pickle")
 
 
+class Test1dsfmAllOutliers(unittest.TestCase):
```

Comment:
why have a new class? We can just add it as a function in the old class?

Reply:
main reason here was to force the case w/ data that would make 1dsfm fail previously